x-pack/test/reporting_api_integration/reporting_and_security/download_csv_dashboard.ts

```diff
@@ -394,6 +394,85 @@ export default function ({ getService }: FtrProviderContext) {
       });
     });
 
+    describe('_id field is a big integer', () => {
```

Comment:
@vadimkibana this is a great point. We also talked offline about how functional tests should be used to enhance a body of unit tests, and that unit test coverage should be greater than functional coverage. I admit I developed a habit of adding functional tests by default, but just for CSV export and not for other features, so that is irrational. 

Going [forward](https://github.com/elastic/kibana/issues/112865), I'll use unit tests and ensure this area of code fits the pattern of greater coverage of unit tests.